vnext/Microsoft.ReactNative/Utils/BatchingEventEmitter.cpp

```diff
@@ -13,18 +13,14 @@ BatchingEventEmitter::BatchingEventEmitter(Mso::CntPtr<const Mso::React::IReactC
   m_uiDispatcher = m_context->Properties().Get(ReactDispatcherHelper::UIDispatcherProperty()).as<IReactDispatcher>();
 }
 
-void BatchingEventEmitter::EmitJSEvent(int64_t tag, winrt::hstring &&eventName, JSValue &&eventObject) noexcept {
-  return EmitJSEvent(L"receiveEvent", tag, std::move(eventName), std::move(eventObject));
+void BatchingEventEmitter::EmitJSEvent(JSValueArgWriter &&paramsWriter) noexcept {
```

Comment:
I'm not sure we should store the delegate for long-term. Other usages have short-term lifetime, so they could do something like pass a lambda capturing by ref that would work there cause memory safety issues here.

Consider taking this in as a const ref and instead storing the serialized dynamic in the `BatchedEvent` struct.

Reply:
I'm going to switch to folly::dynamic.